include/utils/BaselineCorrectionUtils.h

```diff
@@ -0,0 +1,71 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     MASTODON                  */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+// This code was implemented in collaboration with Christopher J. Wong
+// (chris.wong@utah.edu) from the University of Utah.
+
+#pragma once
+
+// LIBMESH includes
+#include "DenseMatrix.h"
+#include "libmesh/dense_vector.h"
+
+/**
+ * This namespace contains the functions used for the calculations corresponding
+ * to the time history adjustment procedure in BaselineCorrection
+ **/
+namespace BaselineCorrectionUtils
+{
+/// Evaluates an integral over a single time step with Newmark-beta method
+/// Also is used as simple trapezoidal rule when gamma = 0.5.
```

Comment:
Consider changing to `Reduces to a simple trapezoidal rule when gamma = 0.5`

Reply:
@somu okay

Reply:
@somu15 okay